pkg/ibmvpc/examples/out/explain_out/GroupingExternalSG1_all_vpcs_explain.txt

```diff
@@ -3,12 +3,12 @@ Interpreted source(s): vsi1-ky[10.240.10.4]
 Interpreted destination(s): 161.26.0.0/8 (external)
 ========================================================================
 
-Connections from vsi1-ky[10.240.10.4] to Public Internet 161.26.0.0/16: protocol: UDP
+Connections from vsi1-ky[10.240.10.4] to Service Network 161.26.0.0/16: protocol: UDP
 
 Path:
 	vsi1-ky[10.240.10.4] -> security group sg1-ky -> network ACL acl1-ky -> subnet subnet1-ky -> 
-	PublicGateway public-gw-ky -> 
-	Public Internet 161.26.0.0/16
+	ServiceNetwork serviceNetwork -> 
```

Comment:
```suggestion
	ServiceGateway -> 
```

Reply:
it should print type and name
it can be 
`ServiceGateway serviceNetwork ->`

Reply:
I prefer just `ServiceGateway` with an empty name.
If it's not possible to have an empty name, let's have something like `ServiceGateway <auto-generated>`

Reply:
fixed